src/modules/core-milestone/core-milestone.tsx

```diff
@@ -0,0 +1,289 @@
+/* eslint-disable no-confusing-arrow */
+
+import * as React from 'react';
+import {IProps} from './core-milestone.types';
+import Icon from '../icon/icon-external';
+import {minorScale} from '../scales';
+import Styled from 'styled-components/native';
+/* @ts-ignore */
+import ShapeHeart from '../../images/shape-heart.svg';
+/* @ts-ignore */
+import Baptism from '../../images/baptism.svg';
+/* @ts-ignore */
+import People from '../../images/people.svg';
+/* @ts-ignore */
+import ServingOpportunity from '../../images/serving-opportunity.svg';
+/* @ts-ignore */
+import ShapeExperiences from '../../images/shape-experiences.svg';
+
+const StyledViewIconWrapper = Styled.View`
+flex-direction: row;
+align-items: center;
+margin-vertical: ${minorScale(1)};
+`;
+
+const StyledEmptyViewWrapper = Styled.View``;
+
+const StyledMilestoneIconWrapper = Styled(Icon)``;
+
+const StyledClassMilestoneWrapper = Styled.View<IProps>`
+flexDirection: row;
+alignItems: center;
+marginRight: ${props =>
+  props.isWhite === true ? `${minorScale(1)}px` : `${minorScale(1)}px`}};
+marginLeft: ${props =>
+  props.isWhite === true ? `${minorScale(1)}px` : `${minorScale(1)}px`}};;
```

Comment:
Please fix the formatting here. Also, do we even need the condition here? because in any case, the margins will be 11px.

Reply:
i will update